framework/graph/GraphArea.kt

```diff
@@ -59,9 +61,9 @@ class GraphArea(transactionState: TransactionState) {
     val coroutines = CoroutineScope(Dispatchers.Default)
     val graphBuilder = GraphBuilder(graph, transactionState, coroutines)
     val viewport = Viewport(graph)
-    val physicsRunner = PhysicsRunner(this)
-    var theme: Color.GraphTheme? = null
     var typography: Typography.Theme? = null
+    private val physicsRunner = PhysicsRunner(this)
+    private val graphTheme = StudioState.preference.graphTheme
```

Comment:
Our preference manager is now the source of truth about what graph theme we use.